### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?
Fixes all the linting and makes sure the build is all valid.

#### Where should the reviewer start?
First pull down the branch ```fix-linking``` and run ```npm install``` to make sure all dependancies are installed.

#### How should this be manually tested?
Once setup run ```gulp test``` and ```gulp build``` and check there are no errors with the SCSS linting.

#### Any background context you want to provide?
_n/a_

#### What are the relevant tickets?
_n/a_

#### Screenshots (if appropriate)
_n/a_

#### Questions:
_n/a_

### CODE REVIEW

**Comment by anthonydillon** [comment]
A lot of this seems to be in https://github.com/ubuntudesign/landscape.canonical.com/pull/83?

**Comment by richmccartney** [comment]
@anthonydillon it used it as a basis as I needed the new updated dependancies, this should be ready to test now with conflicts fixed.

File: static/scss/generic/_tabs.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// scss-lint:disable SelectorFormat NestingDepth
```
Comment (anthonydillon) [inline static/scss/generic/_tabs.scss]: You could tackle the NestingDepth by combining .tab.js-is-active below. Then you will only be 3 levels deep.

Can you add `// convention: 'hyphenated_lowercase'` to the top of the scss file and you shouldn't need to disable SelectorFormat

Comment (richmccartney) [inline static/scss/generic/_tabs.scss]: @anthonydillon Ah this is to be removed as of our design meeting last night.

**Comment by anthonydillon** [comment]
LGTM +1 with addressing the SelectorFormat solution.

**Comment by richmccartney** [comment]
@anthonydillon the linting we want to use is ```convention: 'hyphenated_BEM'```

**Comment by anthonydillon** [comment]
Sorry your right, copied the wrong one.

**Pull request merged**